src/qt/addressbookpage.cpp

```diff
@@ -113,24 +148,12 @@ void AddressBookPage::setModel(AddressTableModel *_model)
     if(!_model)
         return;
 
-    proxyModel = new QSortFilterProxyModel(this);
+    auto type = tab == ReceivingTab ? AddressTableModel::Receive : AddressTableModel::Send;
+    proxyModel = new AddressBookSortFilterProxyModel(type, this);
     proxyModel->setSourceModel(_model);
-    proxyModel->setDynamicSortFilter(true);
-    proxyModel->setSortCaseSensitivity(Qt::CaseInsensitive);
-    proxyModel->setFilterCaseSensitivity(Qt::CaseInsensitive);
-    switch(tab)
```

Comment:
What was this doing originally?

Reply:
This behaviour is now coded in L151.

Reply:
I see. In that case I'm confused as to what `ReceivingTab` does. I only see the address book in the Send tab.

Reply:
Create a request payments in "Receive" tab and then go "File" -> "Receiving adresses...".

Reply:
Found it, search works there as well.